docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Persisted Queries
+---
+
+Apollo supports two separate but related features called **persisted queries** and **automated persisted queries** (APQs).
+With both features, clients can execute a GraphQL operation by sending an operation's ID instead of the entire string. An operation's ID is the SHA256 hash of the full operation string. Querying by ID can significantly reduce latency and bandwidth usage for very large operation strings.
+
+## Differences between persisted queries and APQ
+
+The persisted queries feature requires operations to be preregistered in a **persisted query list** (**PQL**). 
+This allows the PQL to act as a safelist of trusted operations made by your first-party apps. As such, persisted queries is a security feature as much as a performance one.
+
+With APQs, if the server can't find the operation ID the client provides, the server returns an error indicating that it needs the full operation string. If the Apollo iOS client receives this error, it automatically retries the operation with the full operation string.
```

Comment:
> If the Apollo iOS client receives this error, it automatically retries the operation with the full operation string.

I don't remember what the default for APQ behaviour is in Apollo iOS, but this may not be true if it's disabled by default.

Reply:
Ah, good to know. @BobaFetters can you confirm the default behavior?

Will reword if the retries are disabled by default.

Reply:
From what I see in the code it looks like if you have `autoPersistQueries` turned on in the networking then it will automatically retry the request if it fails if the error message contains `PersistedQueryNotFound`, and some other checks like to make sure it hasn't retried already etc

Reply:
Thanks for digging! Is this configurable at all?

Reply:
Doesn't appear to be, looks like if you turn on auto persisted queries the retry based on the `PersistedQueryNotFound` error will always be how this is handled.